Automattic/wp-calypso
# Extract Gutenboarding language picker into @automattic/language-picker

#### Changes proposed in this Pull Request

* Extract the Gutenboarding language picker based on `@wordpress/components` into `@automattic/language-picker` allowing us to use it both in Gutenboarding and in Calypso.
* Replaces the content of the language picker in Gutenboarding with the extracted version.

#### Testing instructions



* Open `/new/language-modal` and confirm that it matches visually and behaviorally with production.
